### PR TITLE
11585 add camel inflected schemas to alternate phone request

### PR DESCRIPTION
**Pull request by erluti**
## Description of change
adding a camel-inflected tests and version of alternate_phone_request

## Original issue(s)
department-of-veterans-affairs/va.gov-team#11585

### CODE REVIEW

File: spec/request/alternate_phone_request_spec.rb [hunk 1]
```diff
@@ -19,6 +20,14 @@
           expect(response).to match_response_schema('phone_number_response')
         end
       end
+      it 'matches the alternate phone schema when camel-inflected' do
+        VCR.use_cassette('evss/pciu/alternate_phone') do
+          get '/v0/profile/alternate_phone', headers: inflection_header
```
Comment (johnpaulashenfelter) [inline spec/request/alternate_phone_request_spec.rb:25]: Shouldn't the headers here be the default headers *and* the X-Key-Inflection? Maybe headers.merge(inflection_header) or similar?

Comment (erluti) [reply spec/request/alternate_phone_request_spec.rb:25]: I was copying the test above it and it doesn't use any headers on its request.  Is there some magic that applies them automatically?

Comment (johnpaulashenfelter) [reply spec/request/alternate_phone_request_spec.rb:25]: i bet they just never got added

Comment (erluti) [reply spec/request/alternate_phone_request_spec.rb:25]: I don't want to create inconsistencies with the parallel tests and it looks like there's a lot of tests in there not using the headers option.

**Pull request merged**